internal/operators/lvm/lvm_operator.go

```diff
@@ -147,56 +171,109 @@ func (o *operator) GetMonitoredOperator() *models.MonitoredOperator {
 }
 
 // GetHostRequirements provides operator's requirements towards the host
-func (o *operator) GetHostRequirements(ctx context.Context, cluster *common.Cluster, _ *models.Host) (*models.ClusterHostRequirementsDetails, error) {
+func (o *operator) GetHostRequirements(ctx context.Context, cluster *common.Cluster, host *models.Host) (*models.ClusterHostRequirementsDetails, error) {
 	log := logutil.FromContext(ctx, o.log)
 	preflightRequirements, err := o.GetPreflightRequirements(ctx, cluster)
 	if err != nil {
-		log.WithError(err).Errorf("Cannot retrieve preflight requirements for cluster %s", cluster.ID)
+		log.WithError(err).Errorf("Cannot retrieve preflight requirements for host %s", host.ID)
 		return nil, err
 	}
+
+	if len(cluster.Hosts) <= 3 {
+		return preflightRequirements.Requirements.Master.Quantitative, nil
+	}
+
+	role := common.GetEffectiveRole(host)
+
+	if role == models.HostRoleWorker && *cluster.HighAvailabilityMode == models.ClusterHighAvailabilityModeFull {
```

Comment:
What is this trying to validate?  What about masters with schedulable masters?  What about SNO+workers?

Reply:
was removed